server/package.json

```diff
@@ -3,15 +3,22 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
-    "start": "node src/index.js"
+    "server": "nodemon",
+    "start": "node index.js"
   },
   "license": "MIT",
   "dependencies": {
+    "dotenv": "^6.2.0",
+    "global": "^4.3.2",
     "graphql-yoga": "^1.17.4",
     "node-dev": "^3.1.3",
-    "prisma-client-lib": "^1.27.4"
+    "prisma": "^1.28.3",
+    "prisma-binding": "^2.3.7",
```

Comment:
Did you actually get binding to work? Did that let you add custom mutation and Queries?